### PR TITLE
Add Gatekeeper support

### PR DESCRIPTION
**Pull request by Mstrodl**
Rust side done. Frontend side done.

### CODE REVIEW

File: onboard/backend/src/api/mod.rs [hunk 1]
```diff
@@ -377,7 +391,7 @@ pub async fn download_game(game_id: String) -> Result<(), Error> {
  * This function will never panic, but contains an `unwrap` call that will never fail. This section
  * is here to make clippy happy.
  */
-pub async fn launch_game(game_id: String) -> Result<JoinHandle<ExitStatus>, Error> {
+pub async fn launch_game(game_id: String) -> Result<(), Error> {
```
Comment (jabbate19) [inline onboard/backend/src/api/mod.rs:394]: Is the handle return not important?

File: onboard/backend/src/servers/onboard.rs [hunk 2]
```diff
@@ -1,186 +1,61 @@
...
+                let mut writer = writer.lock().await;
+                writer.write_all(&response).await?;
+                Ok(()) as Result<(), anyhow::Error>
+            }));
+        }
+        future::join_all(handles).await;
+        Ok(())
+    })
+    .await;
+    panic!("Chom");
```
Comment (jabbate19) [inline onboard/backend/src/servers/onboard.rs]: Can this be something else

**Review by jabbate19 — COMMENTED** [review]
chom

File: onboard/backend/src/api/mod.rs (hunk 1 above)
Comment (Mstrodl) [reply onboard/backend/src/api/mod.rs:394]: It's not used anymore, I refactored that stuff all away

File: onboard/backend/src/servers/onboard.rs (hunk 2 above)
Comment (Mstrodl) [reply onboard/backend/src/servers/onboard.rs]: Done!

**Review by wilsonmcdade — APPROVED** [review]

**Pull request merged**